Frontend/ui-driver/src/Screens/RideSelectionScreen/Controller.purs

```diff
@@ -167,7 +167,7 @@ rideHistoryListTransformer list categoryAction =
     , source : toPropValue (decodeAddress (ride.fromLocation) false)
     , updatedAt  : toPropValue ride.updatedAt
     , driverName : toPropValue ride.driverName
-    , destination  : toPropValue (decodeAddress (ride.toLocation) false)
+    , destination  : toPropValue (maybe "" (\toLocation -> decodeAddress (toLocation) false) ride.toLocation)
```

Comment:
what's the reason for this change?

Reply:
toLocation is now optional field, we don't show tolocation in the ridehistory and invoice for rentals